aws/sdk/integration-tests/using-native-tls-instead-of-rustls/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "using-native-tls-instead-of-rustls"
+version = "0.1.0"
+authors = ["AWS Rust SDK Team <aws-sdk-rust@amazon.com>"]
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dev-dependencies]
+# aws-config pulls in rustls and several other things by default. We have to disable defaults in order to use native-tls
+# and then manually bring the other defaults back
+aws-config = { path = "../../build/aws-sdk/sdk/aws-config", default-features = false, features = [
+  "native-tls",
+  "rt-tokio",
+] }
+# aws-sdk-s3 brings in rustls by default so we disable that in order to use native-tls only
+aws-sdk-s3 = { path = "../../build/aws-sdk/sdk/s3", default-features = false, features = [
+  "native-tls",
+] }
+# aws-sdk-sts is the same as aws-sdk-s3
+aws-sdk-sts = { path = "../../build/aws-sdk/sdk/sts", default-features = false, features = [
+  "native-tls",
+] }
```

Comment:
Is bringing this in required? It's not referenced in the test.

Reply:
Thanks for bringing it up. I think we can remove it. Because the test looks at a dependency tree, I originally thought we needed this as part of the test requirement even though `sts` wasn't used at runtime.